Tests/CommandsTests/APIDiffTests.swift

```diff
@@ -43,63 +50,43 @@ class APIDiffTestCase: CommandsBuildProviderTestCase {
             packagePath,
             extraArgs: args,
             env: environment,
-            buildSystem: buildSystemProvider
+            buildSystem: buildSystem
         )
     }
 
-    func skipIfApiDigesterUnsupportedOrUnset() throws {
-        try skipIfApiDigesterUnsupported()
-        // Opt out from testing the API diff if necessary.
-        // TODO: Cleanup after March 2025.
-        // The opt-in/opt-out mechanism doesn't seem to be used.
-        // It is kept around for abundance of caution. If "why is it needed"
-        // not identified by March 2025, then it is OK to remove it.
-        try XCTSkipIf(
-            Environment.current["SWIFTPM_TEST_API_DIFF_OUTPUT"] == "0",
-            "Env var SWIFTPM_TEST_API_DIFF_OUTPUT is set to skip the API diff tests."
-        )
-    }
-
-    func skipIfApiDigesterUnsupported() throws {
-      // swift-api-digester is required to run tests.
-      guard (try? UserToolchain.default.getSwiftAPIDigester()) != nil else {
-        throw XCTSkip("swift-api-digester unavailable")
-      }
-    }
-
-    func testInvokeAPIDiffDigester() async throws {
-        try skipIfApiDigesterUnsupported()
+    @Test(.requiresAPIDigester, arguments: [BuildSystemProvider.Kind.native, .swiftbuild])
```

Comment:
suggestion: instead of hardcoding `[BuildSystemProvider.Kind.native, .swiftbuild]` we can use `SupportedBuildSystemOnAllPlatforms` (maybe it can be better named)